comictalker/talkerbase.py

```diff
@@ -40,13 +40,15 @@ class SourceStaticOptions:
     def __init__(
         self,
         website: str = "",
+        attribution_string: str = "",  # Website link will be added after this text using source_details.name
```

Comment:
Let's make the attribution_string stand by itself. This string should always make sense by itself. Maybe put a comment that the html anchor tag is understood.